README.md

```diff
@@ -11,6 +11,10 @@ The Temporal CLI is a command-line interface for running Temporal Server and int
 
 ### Installation
 
+#### curl
+
+`curl -sSf https://temporal.download/cli.sh | sh`
```

Comment:
I'm having second thoughts here, maybe call this `install.sh`?
I understand if you don't want to change this yet again.

Reply:
let's discuss in a thread